pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -506,7 +506,7 @@ func convertToUsers(guestInfo *libvirt.DomainGuestInfo) []api.User {
 			users = append(users, api.User{
 				Name:      user.Name,
 				Domain:    user.Domain,
-				LoginTime: float64(user.LoginTime),
+				LoginTime: float64(user.LoginTime) / 1000.0, // ms -> s
```

Comment:
Can you either use `time` to convert or at least add a `const` for `1000.0`?

We don't actually document what this fields provides in our API:

https://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachineinstanceguestosuser

We should also fix this as part of this PR making a note this is time since EPOCH:

https://github.com/qemu/qemu/blob/56c6e249b6988c1b6edc2dd34ebb0f1e570a1365/qga/qapi-schema.json#L1411-L1427